distributed/worker.py

```diff
@@ -385,9 +385,9 @@ class Worker(BaseWorker, ServerNode):
     profile_history: deque[tuple[float, dict[str, Any]]]
     incoming_transfer_log: deque[dict[str, Any]]
     outgoing_transfer_log: deque[dict[str, Any]]
-    incoming_count: int
-    outgoing_count: int
-    outgoing_current_count: int
+    incoming_transfer_count: int
```

Comment:
Instead of talking about `incoming_transfer`/`outgoing_transfer`, which we already had through the `{incoming|outgoing}_transfer_log`, we could also talk about `send`/`recv` to further avoid confusion.

Reply:
This stuff is "public". we should add a deprecation warning (e.g. by using a property). I'm not sure if any downstream project rely on these attributes

Reply:
Personally I think that incoming_count and outgoing_count, which are cumulative since worker start, can be safely dropped without anybody noticing. I doubt they have any usefulness to begin with?